t/30-test_parser.t

```diff
@@ -581,7 +581,34 @@ subtest tap_parse => sub {
     is scalar @{$parser->results->last->details}, 6, '1 test cases details';
 
     is $_->{result}, 'ok', 'All testcases are passing' for @{$parser->results->first->details};
+};
+
+subtest tap_parse_fail => sub {
+    # test other suffix and failing test
+    my $tap_test_file = path($FindBin::Bin, "data")->child("tap_format_example2.tap");
+
+    my $parser = OpenQA::Parser::Format::TAP->new;
+    $parser->load($tap_test_file);
+
+    is $parser->results->size, 1, "One test file";
+    is $parser->results->first->result, 'fail', 'tests failed';
+
+    is scalar @{$parser->results->first->details}, 2, '2 test cases details';
+
+    is $parser->results->first->details->[0]->{result}, 'ok',   'Test 1 passed';
+    is $parser->results->first->details->[1]->{result}, 'fail', 'Test 2 failed';
+};
+
+subtest tap_parse_invalid => sub {
+    # test invalid TAP
+    my $tap_test_file = path($FindBin::Bin, "data")->child("tap_format_example3.tap");
```

Comment:
Also no need for double quotes.

Reply:
The double quotes here are the same as for `tap_format_example.tap`.
What am I supposed to do? use single quotes and be inconsistent?
Or use single quotes and change the original too, although not really related?

Reply:
Hmm, that's annoying. Guess i'm fine with either in that case. 🤷‍♀